app/pages/admin/project-status.jsx

```diff
@@ -258,7 +260,7 @@ class ProjectStatus extends Component {
                 name="project-state-buttons"
                 type="radio"
                 value=""
-                checked={project.state === ''}
+                checked={projectState === 'live' || projectState === 'development'}
```

Comment:
Why is `development` shown as Active? Projects in development aren't listed on the active projects page.

Reply:
@mcbouslog My apologies if my dev notes created any confusion; I didn't mean that `state = 'active'` and `state = 'development'` should be folded together. I was making an observation that projects have at least 4 distinct states, and the fourth `'development'` state isn't shown. (It's fine if it's not shown, because that's a different minor issue for a different minor PR.)

I think this line should remain
```
checked={projectState === 'live'}
```

Reply:
^ yep, pay no attention to https://github.com/zooniverse/Panoptes-Front-End/pull/5292#issuecomment-472054134